public/doc/openapi-3-v1.0.0.json

```diff
@@ -255,11 +255,11 @@
         ],
         "responses": {
           "200": {
-            "description": "Tags collection",
```

Comment:
I think the external documentation should still say "tags".  taggings is an internal concept.

Reply:
I wanted people to know that this is different from the `/tags` endpoint

Reply:
but it is the tags endpoint? `/container_groups/{id}/tags`

Reply:
But the `/tags` primary collection has a different response.  Do we want to change that in this PR as well?

Currently: `GET /api/v1.0/tags`
```
{
  "meta": {
    "count": 2,
    "limit": 100,
    "offset": 0
  },
  "links": {
    "first": "/api/v1.0/tags?offset=0",
    "last": "/api/v1.0/tags?offset=0"
  },
  "data": [
    {
      "created_at": "2019-10-29T19:37:17Z",
      "id": "1",
      "name": "key1",
      "namespace": "bdunne",
      "value": "value1"
    },
    {
      "created_at": "2019-10-29T19:37:26Z",
      "id": "2",
      "name": "key_only_no_value",
      "namespace": "bdunne",
      "value": ""
    }
  ]
}
```

Reply:
Done in this PR, created `/api/v2.0/...`